### ISSUE TITLE
Class Icons for UnitFrames

### ISSUE THREAD

**Issue by Esparanta** [ElvUI-WotLK/ElvUI#839]
# Feature request

Hello. Can I request the feature as titled above? At the moment, class icons can be only seen at target unit frames, not at player/targettarget/focus/party. It's kinda ironic this feature was only implemented at target whereas XPerl, an addon being 10-15 years old, has it for all by default.

**Comment by conscc** [comment]
yes please, thats the only thing i am missing in elvui and would want it so bad 🙏🙏
